src/main/kotlin/no/nav/tilleggsstonader/sak/utbetaling/VentePåStatusFraUtbetalingSteg.kt

```diff
@@ -14,7 +14,8 @@ class VentePåStatusFraUtbetalingSteg(
 
     override fun utførSteg(saksbehandling: Saksbehandling, data: Void?) {
         // TODO sjekk status
-        taskService.save(FerdigstillBehandlingTask.opprettTask(saksbehandling))
+
+        taskService.save(JournalførVedtaksbrevTask.opprettTask(saksbehandling.id))
```

Comment:
Hmm litt usikker på dette med at vi må vente med å journalføre og distribuere brevet før vi setter behandlingen til ferdigstilt.

Mulig vi burde sette status på behandlingen til ferdigstilt her i stedet (?)

Reply:
(Vi kan vurdere dette senere)

Reply:
Tenker det er en funksjonell avveiing. Ønsker vi å kunne opprette revurderinger før brevet er journalført og distribuert? I EF gjør vi ikke det, men kan jo åpne for det her